docker/docker-compose-traefik.yaml

```
@@ -77,17 +95,20 @@ services:
         max-size: "10m"
         max-file: "3"
     command:
-      - --configFile=/config.yaml
+      - "--configFile=/config.yaml"
+      - "--certificatesresolvers.letsEncrypt.acme.email=${ACME_EMAIL:-my.email@gmail.com}"
```

```suggestion
      - "--certificatesresolvers.letsEncrypt.acme.email=${ACME_EMAIL}"
```

Adding a non-working default variable doesn't make sense I think. Does traefik work without an ACME email? That would make more sense I think, to leave out optional entries for now. Especially as the setup refreshes certs on its own anyway.